src/test/java/backend/gameBoard/GameBoardTest.java

```diff
@@ -0,0 +1,18 @@
+package backend.gameBoard;
+
+import backend.logic.Dice;
+import org.junit.jupiter.api.Test;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+class GameBoardTest {
+    @Test
+    void boardSizeToLow(){
```

Comment:
```suggestion
    void boardSizeTooLow(){
```
spelling ;)

but also, this test doesn't do much yet 😝